src/parse.coffee

```diff
@@ -3,10 +3,21 @@ fury.use(require('fury-adapter-apib-parser'))
 fury.use(require('fury-adapter-swagger'))
 
 
-createWarning = (message) ->
-  annotation = new fury.minim.elements.Annotation(message)
-  annotation.classes.push('warning')
-  return annotation
+# Gets the first element that matches the element name by testing the element
+# itself and traversing up through its ancestors in the tree. Works properly
+# only after the element was made immutable by the 'freeze()' method.
+#
+# The name of the method is inspired by https://api.jquery.com/closest/
+fury.minim.Element::closest = (elementName, className = null) ->
```

Comment:
With https://github.com/refractproject/minim/pull/149 this can become:

```js
element.parents.find(elementName)
```

or:

```js
element.parents.find((element) => {
  element.name == elementName && element.classes.contains(className)
})
```